docs/sources/write/shortcodes/index.md

```diff
@@ -547,6 +547,47 @@ Including the original image dimensions as the 'width' and 'height' properties i
 These values are _only_ used for determining the image aspect ratio and don't equate to the final displayed size.
 {{< /admonition >}}
 
+### Center an image example
+
+You can center an image using the `figure` shortcode by adding the following properties:
+
+- Add the `class="w-100p"` property.
+- Add the `link-class="w-fit mx-auto d-flex flex-direction-column"` property.
+- Add the `max-width="WIDTHpx"` property, replacing `WIDTH` with the value you want for your image. The value should be lower than the original image's width.
```

Comment:
"The value should be lower than the original image's width." isn't entirely accurate.
```suggestion
- Add the `max-width="WIDTHpx"` property, replacing `WIDTH` with the value of the maximum width at which your image can be displayed across all breakpoints. This property only affects raster images whose intrinsic (original) width is greater than the property value.
```

Reply:
How about we change it to this:

```suggestion
- Add the `max-width="WIDTHpx"` property, replacing `WIDTH` with the maximum width value you want to display your image. This property only affects raster images, such as PNG or JPG files, whose original width is greater than the property value.
```

I'm thinking if we mention "breakpoints", we should probably include an explanation of what they are as well, and I'm not sure if that's something we need here. What do you think?

Reply:
Works for me!